### PR TITLE
Updated parsing logic and fixed error print statements

### PR DESCRIPTION
**Pull request by RonnieArvanites**
**Summary**
- Updated parsing logic
- Updated print statements to match sample output

### CODE REVIEW

File: GEDCOMReader.py [hunk 1]
```diff
@@ -223,90 +208,68 @@ def age_over_150(indiv):
...
         elif tag == "FAMS":
             spouse = " ".join(components)
+            index = len(individuals) - 1
+            individuals[index].spouse = spouse
         elif tag == "FAMC":
             child = " ".join(components)
-        elif tag == "FAM":
-            family_Id = " ".join(components)
+            index = len(individuals) - 1
+            individuals[index].child = child
```
Comment (Slackow) [inline GEDCOMReader.py]: this can be simplified to individuals[-1].child

File: GEDCOMReader.py [hunk 2]
```diff
@@ -223,90 +208,68 @@ def age_over_150(indiv):
...
         elif tag == "HUSB":
             husband_Id = " ".join(components)
+            husband_name = find_name_for_id(husband_Id)
+            index = len(families) - 1
+            families[index].husband_Id = husband_Id
+            families[index].husband_name = husband_name
```
Comment (Slackow) [inline GEDCOMReader.py]: same here

**Review by Slackow — CHANGES_REQUESTED** [review]
Overall good changes, I think with the individuals[-1] simplification applied it would improve overall, I'd also like to understand the changes in TestFamilyTree.ged, because I see no notes on that.

**Comment by RonnieArvanites** [comment]
> Overall good changes, I think with the individuals[-1] simplification applied it would improve overall, I'd also like to understand the changes in TestFamilyTree.ged, because I see no notes on that.

@Slackow The TestFamilyTree.ged had empty families where there was only one child in the family with no parents. I am not sure why this happened since I exported the file from familyecho.com. To fix this issue, I recreated the family again using the same family names.

**Review by Slackow — APPROVED** [review]

**Review by scolicodes — APPROVED** [review]

**Pull request merged**